test/fixtures/output/network/create-no-name.golden

```diff
@@ -0,0 +1,12 @@
++----------------------------+------------------------------+
+| ID                         | n-abcde1                     |
+| Environment                | env-00000                    |
+| Name                       |                              |
```

Comment:
Optionally you might want to set `omitempty` here.

Reply:
Added `omitempty` here, and for other sub-commands as well.